packages/rs-drive-abci/Cargo.toml

```diff
@@ -26,6 +26,7 @@ serde_with = { version = "3.1.0", features = [
 drive = { path = "../rs-drive", default-features = false, features = [
   "server",
   "grovedb_operations_logging",
+  "grovedbg",
```

Comment:
I believe this should only be on a feature in drive abci right?.

Reply:
could you please rephrase, I'm not following

So it's a feature of GroveDB, then it becomes a feature of drive, or we can require grovedb with it under no condition, but it seems we're going granular inside of Drive